doc/HACKING/EndOfLifeTor.md

```diff
@@ -0,0 +1,52 @@
+
+End of Life on an old release series
+------------------------------------
+
+Here are the steps that the maintainer should take when an old Tor release
+series reaches End of Life:
+
+=== 0. Preliminaries
+
+0. A few months before End of Life:
+   Write a deprecation announcement.
+   Send the announcement out with every new release announcement.
+
+1. A month before End of Life:
+   Send the announcement to tor-announce, tor-talk, tor-relays, and the
+   packagers.
+
+2. A few days before End of Life:
+   Get at least two of weasel/arma/Sebastian to
+   remove the old version number from their approved versions list.
+   Give them a few days to do this if you can.
+
+=== 1. On the day
+
+1. Open tickets to remove the release from:
+   - the jenkins builds
+   - tor's Travis CI cron jobs
+   - chutney's Travis CI tests (#)
+   - stem's Travis CI tests (#)
+
+2. Close the milestone in Trac. To do this, go to Trac, log in,
+   select "Admin" near the top of the screen, then select "Milestones" from
+   the menu on the left.  Click on the milestone for this version, and
+   select the "Completed" checkbox. By convention, we select the date as
+   the End of Life date.
```

Comment:
I would add a step or complement this with "Update wiki page": https://trac.torproject.org/projects/tor/wiki/org/teams/NetworkTeam/CoreTorReleases

Basically, move the milestone down into the EOL table.